src/Framework/MockObject/MockMethod.php

```diff
@@ -124,7 +118,7 @@ public static function fromReflection(\ReflectionMethod $method, bool $callOrigi
             $modifier,
             self::getMethodParameters($method),
             self::getMethodParameters($method, true),
-            $returnType,
+            self::deriveReturnType($method),
```

Comment:
Argument 7 of PHPUnit\Framework\MockObject\MockMethod::__construct cannot be null, possibly null value provided
```
            self::deriveReturnType($method),
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```